js/Stack.js

```diff
@@ -1,5 +1,7 @@
 class Stack {
+    
     arr = [];
+    
```

Comment:
question: Why you have added these empty lines? :)

Reply:
my default editor did it